modules/task/migrate.go

```diff
@@ -5,7 +5,6 @@
 package task
```

Comment:
@zeripath Hello, Are you sure it's necessary to change this file in my pr? Maybe it's not about my pr's topic. Thanks

Reply:
Sorry this is a commit from master I think it you merge in master it will disappear